gossip/src/crds_value.rs

```diff
@@ -132,6 +135,12 @@ impl Sanitize for CrdsData {
             }
             CrdsData::SnapshotHashes(val) => val.sanitize(),
             CrdsData::ContactInfo(node) => node.sanitize(),
+            CrdsData::RestartLastVotedForkSlots(ix, slots) => {
+                if *ix as usize >= MAX_RESTART_LAST_VOTED_FORK_SLOTS as usize {
```

Comment:
Why do you need `as usize` here?
aren't both values of type `EpochSlotsIndex`.

Reply:
Removed.